Weekly Biospecimen CSV Outputs.R

```diff
@@ -6,6 +6,8 @@ library(data.table)
 library(dplyr)
 library(rio)
 library(glue)
+install.packages("openxlsx")
+library(openxlsx)
```

Comment:
I checked the Dockerfile and `openxlsx` is already installed, so I'm assuming this should still run as expected.

Reply:
Great!